articles/build-an-outlook-clone-using-react-hooks/index.md

```diff
@@ -0,0 +1,881 @@
+    },
+    {
+      "id": "15",
+      "body": "<div><p>Aliquam diam mauris, porttitor non commodo at, viverra id dui. Proin finibus a nulla id posuere. Donec ultricies cursus metus, at egestas tortor dignissim eu. Aliquam a massa eu erat gravida ultrices. Vivamus venenatis imperdiet purus, at egestas enim elementum quis. Nunc ultricies sapien sapien, gravida porttitor diam porta non. Fusce pretium sodales erat. Donec interdum ipsum odio, sit amet luctus est pharetra non. Mauris euismod neque eu malesuada scelerisque. In ultricies lectus eu libero accumsan, ultricies commodo nibh consequat. Pellentesque condimentum, neque id sollicitudin egestas, risus est lobortis diam, in faucibus sapien tortor eu felis. Cras a turpis aliquam, tristique eros at, ullamcorper tellus. Maecenas at tortor magna.</p><p>Phasellus eu purus hendrerit, blandit ante ut, rhoncus neque. Nullam mattis non velit nec bibendum. Morbi commodo enim nec semper ultrices. Pellentesque sit amet vestibulum leo. Pellentesque blandit diam in placerat viverra. Phasellus posuere velit mauris, et auctor lectus scelerisque eu. Cras turpis lorem, gravida quis congue id, tristique non lorem. Proin sit amet eros sit amet ligula vehicula faucibus nec quis ipsum. Nullam semper urna sit amet justo iaculis porta. Nullam commodo libero pulvinar, faucibus dui in, viverra ante. Duis vel leo neque.</p><p>Quisque rhoncus dignissim tincidunt. Duis ornare enim pretium imperdiet iaculis. Fusce finibus turpis non lacus convallis vehicula. Quisque et porta orci. Quisque sed erat at diam feugiat viverra. Vestibulum dignissim velit interdum nibh consectetur venenatis. Sed sodales blandit facilisis. Duis elementum, justo at vehicula tempor, libero quam malesuada magna, et fermentum arcu diam vel elit. Pellentesque sollicitudin egestas varius. Vestibulum efficitur tortor eu dolor mollis fringilla. Aliquam tincidunt ornare leo. Pellentesque malesuada urna magna, sed imperdiet leo vehicula eu. In a odio sit amet magna lobortis aliquet a ac est.</p></div>"
+    }
+  ]
+}
+
+```
+
+### Initiate JSON Server
+```
+json-server --watch db.json
+```
+
+If you get an error, do check if **npm** has been added to the environment variables. That should be a quick fix. Add the following to your **PATH** environment variable: C:\Users\YOUR_NAME\AppData\Roaming\npm
+
+Once we have the server running, we can check via Postman or just go to http://localhost:3000/emailbody. To get a specific email body, we can send the *id* as a parameter. The URL is modified as follows: http://localhost:3000/emailbody?id=1. This returns the first email body.
+
+### Let's implement the hooks we just learned
+
+In the earlier article, we have discussed the logic to go about implementing the project from scratch. In this article, we will give more importance to understanding the hooks and performing optimization.
```

Comment:
> In the earlier article,

Throughout this article, it would sound more professional if you said something like, "In my previous article,"

> we have discussed the logic to go about implementing the project from scratch

You should refactor to something like, "we focussed on the logic to go about..."

> In this article, we will give more importance to understanding the hooks and performing optimization.

This should be refactored to something like, "If you need a refresher on how to implement a project from scratch then you should refer back to <insert article title and hyperlink>. In this article, we will focus on understanding the hooks and performing optimization."